populate.rb

```diff
@@ -10,7 +10,7 @@
 require 'octokit'
 
 
-['pdep-utn', 'sagrado-corazon-alcal', 'mumuki', 'arquitecturas-concurrentes'].flat_map do |org|
+['pdep-utn', 'sagrado-corazon-alcal', 'mumuki', 'arquitecturas-concurrentes', 'orga-unq', 'alcal', 'inpr-unq', 'inpr-sarmiento', 'pdp-unsam', 'wollok'].flat_map do |org|
```

Comment:
`alcal` is not a GH organization, just a book (in fact `sagrado-corazon-alcal` is already there).